cis_v140/docs/cis_v140_5_2.md

```diff
@@ -13,8 +13,8 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 3. For each security group to remediate, perform the following:
 4. Select the respective security group
 5. Choose the **Inbound Rules** tab and click **Edit Inbound rules**
-6. Identify the rules to be edited or removed bas on the remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
+6. Identify the rules to be edited or removed based on the remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
 7. Either
-    - A) Update the `source` field to a range other than 0.0.0.0/0 or
+    - A) Update the `source` field to a range other than 0.0.0.0/0 or ::/0
```

Comment:
Hey @akumar-99 , I'm not sure if you were planning on making additional changes, but can you please also revert any changes where you added `::/0`? Afterward, we can accept the PR, thanks!

Reply:
Reverted all the ::/0 changes